app/facades/food_facade.rb

```diff
@@ -0,0 +1,9 @@
+class FoodFacade
+
+  def self.find_food(food)
+    food_finder = FoodService.find_food(food)
+    food_finder[:foods].map do |data|
```

Comment:
You can also retrieve the totalHits here (consider that in Ruby you can return more than one object from a Ruby method)

Reply:
This allows you to not have to save it to each individual PORO but return the data from this method like this:
`[foods, food_finder[:totalHits]]`